sjb/config/test_cases/ami_build_origin_int_rhel_ovn_kubernetes.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
How much time does this save? Handling a separate set of AMIs is a big maintenance burden

Reply:
As much as:
1. Clone openshift origin source
2. Compile openshift origin
3. Build dind images

Can we re-use some AMI that already has openshift origin source in it?

Reply:
The `build` stage of Origin AMIs has latest Origin master built and compiled. Do you want to run against latest? Latest stable?

Reply:
The `build` stage of Origin AMIs has latest Origin master built and compiled. Do you want to run against latest? Latest stable?